### ISSUE TITLE
Project icon

### ISSUE THREAD

**Issue by richardwerkman** [stryker-mutator/mutation-testing-elements#25]
Right now only files and folders are seperated. It would be nice if we could also mark a folder as a project. 

![image](https://user-images.githubusercontent.com/14224500/55626490-f7354b80-57ab-11e9-9d1d-88bbffc149b0.png)

Here we have two projects: `SharedProject` and `TestApp`. Right now it could be confusing to see the projects as folders. Maybe a different icon could help?

**Comment by nicojs** [comment]
So use the directory icon here? Might make more sense indeed...

I don't feel much for introducing  "project" icon. We would need some kind of knowledge of what is a project and what are files/directories.

**Comment by hugo-vrijswijk** [comment]
I don't think we should use the directory icon here. It click through to a file, so it makes sense to be a file

**Comment by nicojs** [comment]
It would be cool to add the "official" file type icons, as vscode does 😍 
![image](https://user-images.githubusercontent.com/1828233/55627713-27cab480-57af-11e9-8f15-236ad0f571e1.png)

But that takes too much time for now (and little return of value 💰 )
